services/brig/src/Brig/Data/MLS/KeyPackage.hs

```diff
@@ -43,30 +46,45 @@ insertKeyPackages uid cid kps = retry x5 . batch $ do
     q :: PrepQuery W (UserId, ClientId, KeyPackageData, KeyPackageRef) ()
     q = "INSERT INTO mls_key_packages (user, client, data, ref) VALUES (?, ?, ?, ?)"
 
-claimKeyPackage :: UserId -> ClientId -> MaybeT (AppIO r) KeyPackageData
-claimKeyPackage u c = MaybeT $ do
+claimKeyPackage :: Local UserId -> ClientId -> MaybeT (AppIO r) (KeyPackageRef, KeyPackageData)
```

Comment:
If possible, it'd be great if you could replace `AppIO r` with `m` for `MonadClient m, MonadReader Env m`.